src/overlays/actors/ovl_Obj_Yasi/z_obj_yasi.c

```diff
@@ -28,25 +30,69 @@ const ActorInit Obj_Yasi_InitVars = {
     (ActorFunc)ObjYasi_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80BB4D90[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneScale, 400, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneDownward, 800, ICHAIN_STOP),
 };
 
-#endif
+void ObjYasi_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ObjYasi* this = THIS;
+    Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
```

Comment:
We like to leave an empty line between the variables declarations and the actual code
```suggestion
    ObjYasi* this = THIS;

    Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
```